functions.php

```diff
@@ -1,8 +1,203 @@
 <?php
 
 add_action( 'wp_enqueue_scripts', 'wsu_alert_enqueue_scripts' );
-
+/**
+ * Enqueues the scripts used by the child theme.
+ */
 function wsu_alert_enqueue_scripts() {
-	wp_enqueue_script( 'wsu-alert-functions', get_stylesheet_directory_uri() . '/js/functions.js', array( 'jquery' ), spine_get_script_version(), true );
-	wp_enqueue_script( 'wsu-alert-tooltip', get_stylesheet_directory_uri() . '/js/dw_tooltip_c.js', array( 'jquery' ), spine_get_script_version(), true );
+	wp_enqueue_script( 'wsu-alert-functions', get_stylesheet_directory_uri() . '/js/functions.js', array( 'jquery' ), spine_get_child_version(), true );
+}
+
+add_filter( 'spine_child_theme_version', 'wsu_alert_theme_version' );
+/**
+ * Provides an updated version number to help in breaking browser cache.
+ *
+ * @since 0.1.0
+ *
+ * @return string
+ */
+function wsu_alert_theme_version() {
+	return '0.0.7';
+}
+
+/**
+ * Provides the current alert level based on whether an active post exists
+ * and what its categorized alert level is.
+ *
+ * @since 0.1.0
+ *
+ * @return string
+ */
+function wsu_alert_level() {
+	$active_alerts = wsu_alert_get_latest();
+
+	if ( 0 === count( $active_alerts ) ) {
+		return 'clear';
+	}
+
+	$first_alert = array_shift( $active_alerts );
+
+	$categories = get_the_category( $first_alert );
+
+	foreach ( $categories as $category ) {
+		if ( in_array( $category->slug, array( 'emergency', 'warning' ), true ) ) {
+			return $category->slug;
+		}
+	}
+
+	return 'clear';
+}
+
+/**
+ * Retrieves the latest active alert.
+ *
+ * @since 0.1.0
+ *
+ * @return array
+ */
+function wsu_alert_get_latest() {
+	$args = array(
+		'meta_key' => 'wsu_alert_status',
+		'meta_value' => 'active',
+		'posts_per_page' => 1,
+		'fields' => 'ids',
+	);
+	$active_alerts = get_posts( $args );
+
+	return $active_alerts;
+}
+
+/**
+ * Displays the header title with an overall alert status based on the current
+ * alert level.
+ *
+ * @since 0.1.0
+ */
+function wsu_alert_display_status_title() {
+	$level = wsu_alert_level();
+
+	if ( 'emergency' === $level ) {
+		echo 'There is an active emergency';
+	} elseif ( 'warning' === $level ) {
+		echo 'There is an active warning';
+	} else {
+		echo 'There are no alerts';
+	}
+}
+
+add_action( 'add_meta_boxes', 'wsu_alert_add_meta_boxes', 10 );
+/**
+ * Adds meta boxes used to manage alerts.
+ *
+ * @since 0.1.0
+ *
+ * @param string $post_type
+ */
+function wsu_alert_add_meta_boxes( $post_type ) {
+	if ( 'post' !== $post_type ) {
+		return;
+	}
+
+	add_meta_box( 'wsu_alert_active_stats', 'Alert Status', 'wsu_alert_display_alert_status', 'post', 'side', 'high' );
```

Comment:
I imagine `wsu_alert_active_stats` is meant to be `wsu_alert_active_status`, but it's not hurting anything the way it is.